doc/source/contributing/issue_triaging.md

```diff
@@ -0,0 +1,24 @@
+# **Issue Triaging**
+
+It is often intimidating to start contributing when you are new and don't know where to start. Or you would like to contribute but don't have the time to code but prefer helping out occasionally with issue triaging. 
+
+The first step for you to get started would be to make sure you subscribe and get notifications of any new issue our team runs into. You can then chose to work on it at the moment or come back to it later. Staying subscribed will ensure you have real time and updated data on where we are with the issue. 
+
+## How to subscribe to issue 
+
+If you are not familiar on how to set up notifications on github, please check the following - [Setting Up Notifications on GitHub](https://docs.github.com/en/github/managing-subscriptions-and-notifications-on-github/setting-up-notifications).
```

Comment:
I would have the first link point to https://docs.github.com/en/github/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications#configuring-your-watch-settings-for-an-individual-repository directly so readers are directed to the most relevant page and section right away. After that we can add extra links with more context and general guidance like you already did on the next paragraph